lib/authentication/secureStorage.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2015-2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+const path = require('path');
+const GlobalConfig = require('../global_config');
+const Logger = require('../logger');
+const keytar = require('keytar');
```

Comment:
just checking this line is fine when we don't have the necessary libs on Linux?

Reply:
I redesigned the secure Storage. Please have a look again.